coins.json

```diff
@@ -292,5 +292,34 @@
 	"min_address_length": 35,
 	"max_address_length": 95,
 	"bitcore": []
+},
+{
+	"coin_name": "Bitcoin Gold",
+	"coin_shortcut": "BTG",
+	"address_type": 38,
+	"address_type_p2sh": 23,
+	"maxfee_kb": 500000,
+	"minfee_kb": 1000,
+	"signed_message_header": "Bitcoin Signed Message:\n",
+	"hash_genesis_block": "000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f",
+	"xpub_magic": "0488b21e",
+	"xprv_magic": "0488ade4",
+	"bip44": 156,
+	"segwit": true,
+	"forkid": 79,
+	"force_bip143": false,
```

Comment:
If Bitcoin Gold decide to use BIP143 for non-SegWit transactions, we need to change this to `true`

Reply:
They may still do it: https://github.com/BTCGPU/BTCGPU/pull/83#discussion_r148296139